src/ast_transform.coffee

```diff
@@ -821,7 +826,7 @@ do ()=>
 
       var_decl.assign_value = arg.clone()
       var_decl.type = mod_decl.type_i.nest_list[idx]
-    if unscoped
+    if not_need_nest
```

Comment:
It'll be clearer to  make `need_nest` and argument by itself and write it like:
`ret.need_nest = need_nest`